lib/core/src/lib/services/discovery-api.service.ts

```diff
@@ -42,7 +42,9 @@ export class DiscoveryApiService {
     ) {
         this.authenticationService.onLogin
             .pipe(
```

Comment:
this PR is for master, should be for develop branch

Reply:
done!